content/en/infrastructure/containers/kubernetes_resource_utilization.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Kubernetes Resource Utilization
+kind: documentation
+---
+
+The Kubernetes resource utilization feature gives you insights into how your Kubernetes workloads are using your computing resources across all your infrastructure. It will help you understand resource usage and make better decisions about sizing and capacity planning, as well as reducing the amount of CPU or Memory waste.
+
+By showing you an always up-to-date state of how well your resource requests and limits track your pods current usage, it will help you increase bin packing in your Kubernetes clusters.
+
+## Getting started
+
+To take advantage of the resource utilization features you'll need to use the Datadog Agent version `7.45.0` or newer. The feature is built on top of the Orchestrator Explorer which also needs to be [enabled][1].
+
+## Usage
+
+Access this page by clicking Infrastructure > Kubernetes in the UI, and selecting the Resource Utilization tab.
```

Comment:
could you add links on:
* `Infrastructure > Kubernetes` to https://app.datadoghq.com/kubernetes
* `Resource Utilization` to https://app.datadoghq.com/orchestration/resource/pod?groups=tag%23kube_deployment%2Ctag%23kube_namespace%2Ctag%23kube_cluster_name